src/Public/js/myradio.core.js

```diff
@@ -49,7 +49,15 @@ var myradio = {
     );
     return reportButton;
   },
-  callAPI: function (method, module, action, id, firstParam, options) {
+  callAPI: function (method, module, action, id, firstParam, options, successFunc = function(){}) {
```

Comment:
Default parameter support is not supported in Safari < 10, or any version Opera. (or IE, but meh). Discuss whether this is acceptable minimum browser support and change if not.